src/content/docs/new-relic-solutions/best-practices-guides/full-stack-observability/synthetic-monitoring-best-practices-guide.mdx

```diff
@@ -13,115 +13,130 @@ redirects:
+            openssl s_client -servername {YOUR_HOSTNAME} -connect {YOUR_HOSTNAME}:443 -CApath /etc/ssl/certs > /dev/null
+            ```
 
-4. Select the locations from where you want your monitor to run. We recommend choosing at least three locations to avoid false positives. In other words, if at least one location returns a successful result then the endpoint must be up and triggering of an alert can be avoided.
-5. Depending on the monitor type, you'll be prompted to either <DoNotTranslate>**Save monitor**</DoNotTranslate>, <DoNotTranslate>**Validate**</DoNotTranslate>, or <DoNotTranslate>**Write script**</DoNotTranslate>.
-6. View your results as they're received in the [summary page](#summary-page).
+        * <DoNotTranslate>**Bypass HEAD request (for ping monitors).**</DoNotTranslate> This option skips the default HEAD request and instead uses the GET verb with a ping check. GET requests will always happen if the HEAD request fails.
+        * <DoNotTranslate>**Redirect is Failure (for ping).**</DoNotTranslate> If a redirect result occurs when `Redirect is Failure` is enabled, your synthetic monitors categorize it as a failure rather than following the redirect and checking the resulting URL.
 
-## 2. View the summary page for your synthetic monitors [#summary-page]
+    5. Select the locations from where you want your monitor to run. We recommend choosing at least three locations to avoid false positives. In other words, if at least one location returns a successful result then the endpoint must be up and triggering of an alert can be avoided.
+    6. Depending on the monitor type, you'll be prompted to either <DoNotTranslate>**Save monitor**</DoNotTranslate>, <DoNotTranslate>**Validate**</DoNotTranslate>, or <DoNotTranslate>**Write script**</DoNotTranslate>.
+    7. View your results as they're received in the [summary page](#summary-page).
 
-On the summary page, you'll see information about the status of your synthetic monitor. If something created an active incident that triggered an alert, then click the critical alert for this monitor to open in a new tab. You can also access all your alert policies for your monitors when you click to manage alert policies for all monitors at the top of the page.
+    </Step>
 
-<img
-  src={syntheticsMonitorCriticalAlert}
-  title="Synthetic monitor critical alert"
-  alt="A screenshot that shows where your critical alerts are located in the UI."
-/>
+    <Step>
+    ## View the summary page for your synthetic monitors [#summary-page]
 
-## 3. Get an overview of your application's performance [#synthetics-monitors]
+    On the summary page, you'll see information about the status of your synthetic monitor. If something created an active incident that triggered an alert, then click the critical alert for this monitor to open in a new tab. You can also access all your alert policies for your monitors when you click to manage alert policies for all monitors at the top of the page.
```

Comment:
"to open it in a new tab"

for the last sentence, I'm not sure what "when you click to manage" means. Click the alert, click something after you've clicked the critical alert? I think juts a bit more clarification would be helpful.

Reply:
Fixed it.